hoomd/operations.py

```diff
@@ -117,20 +197,50 @@ def integrator(self, op):
 
     @property
     def updaters(self):
+        """list[``Updater``]: A list of updater operations.
```

Comment:
The actual base class is private, so I was avoiding linking to an actual class.

And yes.

Reply:
We use a lot of classes that users should never instantiate, and we normally make them private to signify that. However, they have to be exposed to the user for them to use. We could make the classes public, but then we would need to warn that they should never be instantiated.

Reply:
I do like the `List[_Updater]` it matches Python's own typing system, but sphinx does not know how to properly link to the standard Python documentation. Though, this is a decision we should discuss.

The reason I choose not to document the actual type here is that it appears as an ordinary list to users (besides the type of course). The other private classes we document are to prevent redocumenting a large number of methods which isn't the case here.  We could document `_Updaters` and the like regardless.